TrustModel_Personas.md

```diff
@@ -0,0 +1,195 @@
+# Trust Model Considerations - Personas
+
+## Personas
+Otherwise referred to as actors or threat agents, these are individuals or groups capable of 
+carrying out a particular threat. 
+In identifying personas we consider :
+- The Runtime Environment, Figure 5, Page 19 of CNCF 
+  [Cloud Native Security Paper](https://github.com/cncf/tag-security/blob/main/security-whitepaper/v2/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf).
+  This highlights three layers, Cloud/Environment, Workload Orchestration, Application.
+- The Kubernetes 
+  [Overview of Cloud Native Security](https://kubernetes.io/docs/concepts/security/overview/)
+  identifies the 4C's of Cloud Native 
+  Security as Cloud, Cluster, Container and Code. However data is core to confidential 
+  containers rather than code.
+- The Confidential Computing Consortium paper 
+  [A Technical Analysis of Confidential Computing v1.1](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/CCC-Tech-Analysis-Confidential-Computing-V1.pdf)
+ defines Confidential Computing as the protection of data in use by performing computations in a 
+  hardware-based Trusted Execution Environment (TEE).
```

Comment:
From our conversation outside the PR I think we are surfacing a key difference that any Trust Model only works in the context of its Personas. 
And that since we are bringing the world of Trusted Execution Environments together with the world of Cloud Native we have two ways of viewing the Trust Model. This PR is very much intended to be on the path to Cloud Native Trust Model but I can see that another Trust Model could be more TEE focused and consider host owner, guest owner and end user as personas.
A TEE Focused Trust Model can certainly capture our initial conversations which came from the early discussions on the technology we are using. Perhaps this helps with initial development?
Whereas the Cloud Native Trust Model is less clear, requires more conversation but once there, will be the thing which sees us fit seamlessly into the Cloud Native ecosystem and gains us cloud native adopters/users.

I believe that Confidential Containers can improve the Confidential Computing Cloud Native Story, but I do not expect us to solve everything that a Cloud Native trust Model might highlight. Rather we improve on what exists today, identifying in cloud native terms the threats we can solve, the threats we cannot solve alone and the extensions to cloud native security and practices that should be in place to correctly use confidential containers to improve the conifdential computing story

Would it make sense to revisit [#28] and identify the need for a document which captures a TEE focused trust model?